OptionSelector/Categories.h

```diff
@@ -0,0 +1,17 @@
+//
+//  Categories.h
+//  OptionSelector
+//
+//  Created by Charles Kang on 8/11/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface Categories : NSObject
```

Comment:
Classes nearly always should have singular names. This is a single category, not a set of categories.

(Even classes that hold multiple things are generally singular; "Set", "List", "Deck", etc.)